.gitignore

```diff
@@ -0,0 +1,43 @@
+MANIFEST
```

Comment:
Why do we need .gitignore here?

Reply:
Mainly, it was due to prevent pushing .ipynb checkpoints from my local instance of jupyter notebook, that is why I added the .gitignore.